src/SettingsForm.tsx

```diff
@@ -27,12 +27,14 @@ export const SettingsForm = () => {
   const { isRendering } = useContext(ImagesContext);
 
   const handleChange = useCallback(
-    (key: keyof typeof settings, eventKey: "value" | "checked" = "value") =>
-      (e: React.ChangeEvent<HTMLInputElement>) => {
+    (key: keyof typeof settings, eventKey: "value" | "select" | "checked" = "value") =>
```

Comment:
it doesn't look like we need the updates to the `eventKey` as we are not using it below.
```suggestion
    (key: keyof typeof settings, eventKey: "value" | "checked" = "value") =>
```